api/lib/reducers/cosmosV0-reducers.js

```diff
@@ -233,6 +234,7 @@ function validatorReducer(networkId, signedBlocksWindow, validator) {
 
 function blockReducer(networkId, block, transactions, data = {}) {
   return {
+    id: block.block_meta.header.height,
```

Comment:
that would conflict between different chains, right?
```suggestion
    id: `${networkId}_${block.block_meta.header.height}`,
```

Reply:
or use block hash?

Reply:
:thinking: maybe better yes